Source/WebCore/history/ProcessSwapDisposition.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2006, 2008, 2011, 2014 Apple Inc. All rights reserved.
+ * Copyright (C) 2012 Research In Motion Limited. All rights reserved.
```

Comment:
Should this be updated?

Reply:
I don't think so since I'm just moving the code from one place to another.